src/main/java/uk/org/openbanking/datamodel/payment/OBWriteDomesticStandingOrder3.java

```diff
@@ -46,12 +46,12 @@
 @javax.annotation.Generated(value = "io.swagger.codegen.languages.JavaClientCodegen", date = "2019-05-23T11:45:43.255+01:00")
 public class OBWriteDomesticStandingOrder3 {
     @JsonProperty("Data")
-    private OBWriteDataDomesticStandingOrder3 data = null;
+    private OBWriteDomesticStandingOrder3Data data = null;
```

Comment:
I'm surprised by this, especially if it's a breaking change to the resulting JSON. Normally they would introduce a new enclosing object (e.g. `OBWriteDomesticStandingOrder4`).

Either I made a mistake or the spec I had was very provisional!